ci/workflows/smoke.workflow

```diff
@@ -0,0 +1,39 @@
+workflow "smoke tests" {
+  resolves = "run test"
+}
+
+# NOTE:
+# - the skyhook-test-data image has nothing but the data
+# - the ceph-builder image is a pre-built github action
+#   that is available at https://github.com/popperized/ceph
+
+action "build ceph" {
+  uses = "docker://popperized/ceph-builder:nautilus"
```

Comment:
let's use the local install_deps.sh file from skyhook instead of mainline ceph. since we update that file whenever we add new file access libraries.  thanks.

Reply:
this is the most important fix, since our arrow-dev PR depends upon using the skyhook install-deps.sh insted of ceph's.